Match mode-line colors to Atom status-bar

jhacksworth opened the pull request:
I thought it would be nice if the mode-line colors matched the Atom status-bar.

This commit:

- Matches the colors of the active mode-line to the status-bar by
  updating `atom-one-dark-silver` and `atom-one-dark-black` to values
  found by inspecting the Atom editor.

- Adds a border (in GUI Emacs) around the active mode-line,
  matched to the border color of the Atom status-bar.

- Helps to differentiate between the active and inactive
  mode-lines by dimming the foreground color of the inactive mode-line.

I used a hard-coded value for the color of the active mode-line box border, since I don't see much use for it in other places. If you prefer, it could be defined as `atom-one-dark-border` or whatever you think makes sense.

jhacksworth committed:
Match mode-line colors to Atom status-bar

The Atom status-bar is the counterpart to Emacs' mode-line.

This commit:

- Matches the colors of the active mode-line to the status-bar by
  updating "atom-one-dark-silver" and "atom-one-dark-black" to values
  found by inspecting the Atom editor.

- Adds a border (in GUI Emacs) around the active mode-line,
  matched to the border color of the Atom status-bar.

- Helps to differentiate between the active and inactive
  mode-lines by dimming the foreground color of the inactive mode-line.

jonathanchu commented:
Hey @jhacksworth!  Thanks for this!  I think I would prefer it to be defined as `atom-one-dark-border` or whatever you see fit, as even though it's only used in one place now, at least it will be ready for re-use in the future :) What do you think?

If agreeable, please do make that tiny edit and I'll be happy to Just Merge™ :beers:

jhacksworth committed:
Add `atom-one-dark-border` color variable

Define a new color variable, `atom-one-dark-border`, instead of using a
hard-coded value for the border in the mode-line definition.

jhacksworth commented:
Hi @jonathanchu! Thanks for merging the previous pull requests 😄 I committed the change you suggested, thanks for taking the time to look it over. Cheers! 🍻

jonathanchu reviewed (approved):
💥

jonathanchu commented:
Hey @jhacksworth!  Thanks for the fast turnaround and all your great work on this to make it even better :)
